includes/interfaces/class-wc-importer-interface.php

```diff
@@ -67,6 +67,14 @@ public function get_parsed_data();
 	 */
 	public function get_file_position();
 
+	/**
+	 * Get the size of the file to be imported
+	 * The import is complete when get_file_position() equals get_file_size()
+	 *
+	 * @return int
+	 */
+	public function get_file_size();
```

Comment:
Adding this method into this interface will cause issues for everything that currently is implementing this interface:

```
PHP Fatal error:  Class CLASS_NAME contains 8 abstract method and must therefore be declared abstract or implement the remaining methods (WC_Importer_Interface::get_file_size) in test.php on line 11
```

Reply:
My implementation follows the suggestion by @vedanshujain (https://github.com/woocommerce/woocommerce/issues/29793#issuecomment-831300519).
In the linked issue I also suggest a different, less invasive, fix.

I'll leave the decision how to proceed up to you as I have significantly less experience with he WooCommerce codebase.

Reply:
I don't think we should to add this method in the interface, since it will cause issues for every class that implements this interface. Maybe we can move this method up?